.gitignore

```diff
@@ -0,0 +1,6 @@
+*.class
+*.jar
```

Comment:
Please don't check in gitignore, it becomes a massive mess of everyone's personal coding environment quirks. Better for everyone to just keep their own locally.

Reply:
I'll remove it.